h/static/scripts/group-forms/components/forms/Checkbox.tsx

```diff
@@ -0,0 +1,48 @@
+import type { CheckboxProps as BaseCheckboxProps } from '@hypothesis/frontend-shared';
+import { Checkbox as BaseCheckbox } from '@hypothesis/frontend-shared';
+import type { ComponentChildren } from 'preact';
+import { useId } from 'preact/hooks';
+
+export type CheckboxProps = Omit<BaseCheckboxProps, 'aria-describedby'> & {
+  /**
+   * Adds a description right under the checkbox and main content, aligned with
+   * the main content's left side.
+   */
+  description?: ComponentChildren;
+
+  'aria-describedby'?: string;
+};
+
+export default function Checkbox({
```

Comment:
Can you add a JSDoc comment to summarize what this is for, as in, how it differs from the pattern library checkbox.